includes/Modules/TagManager.php

```diff
@@ -431,6 +428,42 @@ protected function create_data_request( $method, $datapoint, array $data = array
 		return new WP_Error( 'invalid_datapoint', __( 'Invalid datapoint.', 'google-site-kit' ) );
 	}
 
+	/**
+	 * Creates GTM Container.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param string $account_id  The account ID.
+	 * @return mixed Container ID on success, or WP_Error on failure.
+	 */
+	protected function create_container( $account_id ) {
+		$client     = $this->get_client();
+		$orig_defer = $client->shouldDefer();
+
+		$client->setDefer( false );
+
+		$container = new \Google_Service_TagManager_Container();
+		$container->setName( remove_accents( get_bloginfo( 'name' ) ) );
+		$container->setUsageContext( array( 'web' ) );
+
+		try {
+			$container = $this->get_service( 'tagmanager' )->accounts_containers->create( "accounts/{$account_id}", $container );
+		} catch ( Google_Service_Exception $e ) {
+			$client->setDefer( $orig_defer );
+			$message = $e->getErrors();
+			if ( isset( $message[0] ) && isset( $message[0]['message'] ) ) {
```

Comment:
Should be able to simplify this to `if ( isset( $message[0]['message'] ) )` without throwing any warnings or anything.